Shoko.Server/API/v3/Models/Shoko/Episode.cs

```diff
@@ -251,4 +304,117 @@ public class EpisodeIDs : IDs
             #endregion
         }
     }
+
+    public interface IEpisodeMetadata
```

Comment:
If you are splitting out an interface, put it in its own file. Probably, API/v3/Interfaces/Shoko/IEpisode.cs

But also, what's the point here? There won't ever be extended models based on these interfaces aside from the one that already exists.

Reply:
Removed the interfaces, and reverted most of the cosmetic changes.